lib/utils.dart

```diff
@@ -0,0 +1,62 @@
+import 'dart:io';
+
+import 'package:args/args.dart';
+import 'package:prompts/prompts.dart' as prompts;
+import 'package:io/ansi.dart' as ansi;
+
+String getString(
+  ArgResults results,
+  bool isInteractive,
+  String name,
+  String message, {
+  String desc,
+}) {
+  var value = results[name] as String;
+  if (!isInteractive) {
+    if (value == null || value.isEmpty) {
+      print('Missing parameter $name is required.');
+      exit(1);
+    }
+  }
+  while (value == null || value.isEmpty) {
+    if (desc != null) {
+      stdout.write(ansi.darkGray.wrap('\n$desc\u{1B}[1A\r'));
+    }
+    value = prompts.get(message, validate: (it) => !it.contains('-'));
+    if (desc != null) {
+      stdout.write('\r\u{1B}[K');
+    }
+  }
+  return value;
+}
+
+String getOption(
```

Comment:
Maybe we could make use of this package here?
https://pub.dev/packages/args

Reply:
wdym? I use that package!